vita3k/renderer/src/vulkan/pipeline_cache.cpp

```diff
@@ -478,9 +478,14 @@ vk::PipelineVertexInputStateCreateInfo PipelineCache::get_vertex_input_state(Mem
 
         const bool is_instanced = gxm::is_stream_instancing(static_cast<SceGxmIndexSource>(stream.indexSource));
 
+#ifdef __APPLE__
+        const uint32_t stride = (stream.stride + 3) & -4;
```

Comment:
```suggestion
        const uint32_t stride = align(stream.stride, 4);
```
You may need to include `util/align.h`